server/services/involvedStaffService.ts

```diff
@@ -81,7 +81,7 @@ export class InvolvedStaffService {
     })
   }
 
-  public removeInvolvedStaff = async (reportId: number, statementId: number): Promise<void> => {
+  public async removeInvolvedStaff(reportId: number, statementId: number): Promise<void> {
```

Comment:
jest auto mocking doesn't work for class properties that are functions - only methods